.github/workflows/publish_conda.yml

```diff
@@ -7,12 +7,15 @@ jobs:
   publish:
     runs-on: ubuntu-latest
     container:
-      image: continuumio/miniconda3:latest
+      image: ghcr.io/noaa-gfdl/fre-cli:miniconda24.7.1_gcc14.2.0
     steps:
       - name: Checkout Files
         uses: actions/checkout@v4
         with:
           submodules: 'recursive'
+      - name: Add mkmf to PATH
```

Comment:
Oooo, do we need to add this for submodules? The submodules wouldn't just be built with the conda package with the `submodules: 'recursive'`?